utils/ort/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -35,9 +35,9 @@ class CopyrightStatementsProcessorTest : WordSpec({
             val statements = File("src/test/assets/copyright-statements.txt").readLines()
             val expectedResult = File("src/test/assets/copyright-statements-expected-output.yml").readText()
 
-            val actualResult = CopyrightStatementsProcessor.process(statements.shuffled()).toYaml()
+            val result = CopyrightStatementsProcessor.process(statements.shuffled()).toYaml()
 
-            actualResult shouldBe expectedResult
+            result shouldBe expectedResult
```

Comment:
Just a remark, not a blocker: Personally I liked this to explicitly read "*actual*Result shouldBe *expected*Result", also because it matches the terminology used in IntelliJ's diff viewer.